packages/react/src/components/SearchBox/SearchBox.base.tsx

```diff
@@ -125,8 +125,18 @@ export const SearchBoxBase: React.FunctionComponent<ISearchBoxProps> = React.for
     [customOnBlur],
   );
 
+  let _lastChangeValue: string | undefined;
+
   const onInputChange = (ev: React.ChangeEvent<HTMLInputElement>) => {
-    setValue(ev.target.value);
+    const value = ev.target.value;
```

Comment:
This is causing a lint error since `value` is already declared in the upper scope
https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=196583&view=logs&j=258ec178-2d8b-5611-7b9b-60c5c95dae55&t=cc94c182-d6f4-5dae-734d-3676c12cee2a&l=3169